src/services/TestResultsService.js

```diff
@@ -144,8 +144,9 @@ class TestResultsService {
         if (error.statusCode === 400 && error.message === 'The conditional request failed') {
           console.log('Error in insertTestResult > getTestTypesWithTestCodesAndClassification: Test Result id already exists', error)
           return Promise.reject(new HTTPResponse(201, 'Test Result id already exists'))
+        } else if (error.statusCode === 404 && error.body === 'No resources match the search criteria.') {
+          return Promise.reject(new HTTPResponse(404, 'Test types not found'))
         }
-        console.log('Error in insertTestResult > getTestTypesWithTestCodesAndClassification', error)
```

Comment:
why was this removed? It was specifically added only a few weeks ago.

Reply:
By mistake. I've put it back